### ISSUE TITLE
DCI HDR Question

### ISSUE THREAD

**Issue by SteveLLamb** [ISDCF/registries#974]
I do have a lot of questions about the DCI's HDR standard because all I know is that the peak white is 300 nits, the black level is 0.005 nits, the contrast ratio is 60,000:1, the 12-bit-per-color (36-bit-per-pixel) precision will still be used, and the DCI-P3 and Rec. 2020 color spaces will still be used.

SDR and Dolby Vision DCPs are encoded within the CIE XYZ color primaries while Eclair Color DCPs are encoded within the wider ACES AP1 color primaries.  What color primaries will DCPs in the DCI’s HDR standard will be encoded within?

Has the image codec changed from JPEG 2000 in the DCI’s SDR standard?  If so, what is the new codec?  If not, which version of JPEG 2000 is being used?  Regular or High Throughput?

Has the maximum bitrate changed?  SDR and Eclair Color DCPs have a maximum bitrate of 250 megabits per second while Dolby Vision DCPs have a maximum bitrate of 500 megabits per second.  Note that this is for 2D at 24 frames per second.

Has the minimum digital input changed from a dual HD-SDI?  I believe it has changed to a more capable one.

_Originally posted by @jackers2024 in https://github.com/ISDCF/registries/issues/941#issuecomment-2366831239_

**Comment by SteveLLamb** [comment]
Full discussion:
https://github.com/ISDCF/registries/issues/941#issuecomment-2366831239
https://github.com/ISDCF/registries/issues/941#issuecomment-2366834612
https://github.com/ISDCF/registries/issues/941#issuecomment-2366835722
https://github.com/ISDCF/registries/issues/941#issuecomment-2366902185
https://github.com/ISDCF/registries/issues/941#issuecomment-2366908411
https://github.com/ISDCF/registries/issues/941#issuecomment-2366919115
https://github.com/ISDCF/registries/issues/941#issuecomment-2366932266
https://github.com/ISDCF/registries/issues/941#issuecomment-2366932411
https://github.com/ISDCF/registries/issues/941#issuecomment-2366976244
https://github.com/ISDCF/registries/issues/941#issuecomment-2366979079

As noted in https://github.com/ISDCF/registries/issues/941#issuecomment-2369202568, this discussion may continue here if needed, will be closed if resolved.
